sktime/forecasting/conditional_invertible_neural_network.py

```diff
@@ -208,7 +208,19 @@ def _fit(self, y, fh, X=None):
         Returns
         -------
         self : reference to self
+        Common Errors:
+        --------------
+        1. If `window_size` > len(y_train), an error will be raised.
+        2. If curve fitting fails, ensure your data and window_size are suitable.
+
         """
+        # Ensure that window_size is smaller than the training data
+        forecaster = CINNForecaster(
+            window_size=20
```

Comment:
And you are doing that with a fixed value, instead of using user input to the init method. Why?